expectMessage/matchers.ts

```diff
@@ -0,0 +1,78 @@
+import { colorize } from "./utils/colorizeLog";
+
+class JestAssertionError extends Error {
+  matcherResult: any;
+
+  constructor(result: any, callsite: any) {
+    super(result.message());
+    this.matcherResult = result;
+
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, callsite);
+    }
+  }
+}
+
+interface TerminalCustomStyle {
+  fg?: string;
+  bg?: string;
+  effects?: string;
+}
```

Comment:
We could export this one from `./utils/colorizeLog` and import it there instead of redefining it